exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -90,7 +90,8 @@ func TestTelemetryEnabled(t *testing.T) {
 	assert.EqualValues(t, 1, sink.StopCount.Load())
 	assert.True(t, sink.HasRecording())
 	got := sink.Rotate()
-	assert.EqualValues(t, 0, *got.BackendConnectionErrors.HTTPCode4XXCount)
+	assert.EqualValues(t, 1, *got.BackendConnectionErrors.HTTPCode4XXCount)
```

Comment:
BackendConnectionErrors.OtherCount was 1 in version 1, but HTTPCode4XXCount was 1 more after v2 conversion